packages/docusaurus/src/webpack/utils.ts

```diff
@@ -203,15 +203,15 @@ export function compile(config: Configuration[]): Promise<void> {
     const compiler = webpack(config);
     compiler.run((err, stats) => {
       if (err) {
-        reject(err);
+        reject(new Error(err.toString()));
```

Comment:
Was wondering, shouldn't we use an early return here?  
Or are there cases where we have both an error and stats?

Reply:
I don't know, I think we'll leave it as it is for now, during the migration to webpack 5, let's get back to this.